apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/actions.ts

```
@@ -131,5 +132,12 @@ export const updateSurveyAction = authenticatedActionClient
       await checkMultiLanguagePermission(organizationId);
     }
 
+    if (parsedInput.recaptcha && parsedInput.recaptcha.enabled) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)